core/templates/domain/topic/topic-update.service.ts

```diff
@@ -795,6 +795,35 @@ export class TopicUpdateService {
           newSkillId, skillSummary.getDescription());
       });
   }
+
+  /**
+   * Update the skill ids for the diagnostic test from a topic and records
+   * the change in the undo/redo service.
+   */
+  updateDiagnosticTestSkills(
+      topic: Topic, newSkillsForDiagnosticTest: ShortSkillSummary[]): void {
+    let oldSkillsForDiagnosticTest = cloneDeep(
```

Comment:
Naming here may need to be updated (skill --> skill summary).

Possibly elsewhere too -- please check.

Reply:
Updated